packages/sdk/python/human-protocol-sdk/human_protocol_sdk/escrow.py

```diff
@@ -900,11 +900,39 @@ def get_escrows(
                     "to": int(filter.date_to.timestamp()) if filter.date_to else None,
                 },
             )
-            escrows = escrows_data["data"]["escrows"]
-            for escrow in escrows:
-                escrow["chain_id"] = chain_id
-            escrow_addresses.extend(escrows)
-        return escrow_addresses
+            escrows_raw = escrows_data["data"]["escrows"]
+
+            escrows.extend(
+                [
+                    EscrowData(
+                        id=escrow["id"],
```

Comment:
Do we have a guarantee that all the fields we are trying to access will be presented in a `escrow` dict?
Anyways, I think it's not safe to access fields in this way because it can cause a `KeyError`, better to use `.get()`.
https://docs.python.org/3/library/stdtypes.html#dict.get